src/fswAlgorithms/attControl/thrMomentumManagementCpp/_UnitTest/test_thrMomentumManagementCpp_integrated.py

```diff
@@ -0,0 +1,283 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2024, Laboratory of Atmospheric and Space Physics, University of Colorado at Boulder
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+
+import os
+import pytest
+import numpy as np
+import matplotlib.pyplot as plt
+
+from Basilisk import __path__
+from Basilisk.simulation import spacecraft, reactionWheelStateEffector, extForceTorque, simpleNav
+from Basilisk.fswAlgorithms import thrMomentumManagementCpp, inertial3D, attTrackingError, mrpFeedback, rwMotorTorque
+from Basilisk.utilities import SimulationBaseClass, macros, fswSetupRW, simIncludeRW, unitTestSupport
+from Basilisk.architecture import messaging
+
+bskPath = __path__[0]
+fileName = os.path.basename(os.path.splitext(__file__)[0])
+
+
+@pytest.mark.parametrize("hd_B", [np.array([110, 50, -10])])
```

Comment:
:small_red_triangle_down: One value, so doesn't need to be parameterized.

Reply:
Done.